internal/alert/manager.go

```diff
@@ -135,13 +136,36 @@ func (am *alertManager) handlePagerDutyPost(alert core.Alert) error {
 			return fmt.Errorf("client %s could not post to pagerduty: %s", pdc.GetName(), resp.Message)
 		}
 
-		am.logger.Debug("Successfully posted to ", zap.Any("resp", resp))
+		am.logger.Debug("Successfully posted to PagerDuty", zap.Any("resp", resp))
 		am.metrics.RecordAlertGenerated(alert, core.PagerDuty, pdc.GetName())
 	}
 
 	return nil
 }
 
+func (am *alertManager) handleSNSPublish(alert core.Alert, policy *core.AlertPolicy) error {
+	event := &client.AlertEventTrigger{
+		Message:  am.interpolator.SlackMessage(alert, policy.Msg),
```

Comment:
The messages we post to slack are unstructured and contain emojis. Perhaps a JSON structured payload would be better fit for an sns subscriber to reason with?

Reply:
Added a quick unit test to document how the message is marshalled/unmarshalled since aws requires some odd nesting